weatherapplication.client/src/app/app.module.ts

```diff
@@ -19,18 +20,23 @@ import { FormsModule, ReactiveFormsModule } from '@angular/forms';
     CurrentWeatherPageComponent,
     FivedaysWeatherPageComponent,
```

Comment:
I don't know such a word, `Fivedays`, seems to be a typo.

Reply:
![image](https://github.com/Oleksandr-Ryabishchuk/WeatherApplication/assets/36698610/9fccd0d5-c7dd-4571-8f7e-6ae4ebb2d0d7)